CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 ## Changelog
 
 ## 0.15.10
-* [#560](https://github.com/awslabs/amazon-kinesis-producer/pull/560)Reverting CAA related changes
-  * Revert all the changes related to CAA- reverting streamArn after this update customers will not be able to use it until there is a new release with appropriate fixes.
+* [#560 ](https://github.com/awslabs/amazon-kinesis-producer/pull/560) Reverting CAA related changes
```

Comment:
Let's remove "Reverting CAA related changes", as CAA is not a terminology our customers might understand and the specific change is related to removing Stream ARN.